nodes/modifier_change/triangulate.py

```diff
@@ -125,6 +126,8 @@ def process(self):
                 new_face_data = []
             bm.free()
 
+            new_faces = get_unique_faces(new_faces)
```

Comment:
@portnov why did you add this line?, it slows a lot the node

Reply:
I don't remember exactly, there was some node which created meshes with duplicate faces, which broke the node...

Reply:
So you want me to leave it as an option or just delete it?

Reply:
Probably the correct option would be to remove it from here, and to create a separate node for "fixing any mesh"...

Reply:
> Probably the correct option would be to remove it from here, and to create a separate node for "fixing any mesh"...

Yes, there is even request for such node. #3989

Reply:
i think such node might want to also output the indices from the original face list, which it has discarded

Reply:
I thought it should remove only repeated faces and edges. Bmesh module should be able to handle repeated vertices (with the same coordinates). Such vertices can be removed by 'Merge by distance' if needed.